### ISSUE TITLE
nixops: containers are incompatible with unstable nixos-container.pl

### ISSUE THREAD

**Issue by adrianparvino** [NixOS/nixpkgs#72407]
**Describe the bug**
NixOps is unable to `cat /etc/os-release`, stuck at
```
container0> IP address is 10.233.5.2
```

**To Reproduce**
1. Create the following NixOps file:

containers.nix
```nix
{
  container0 = {
    deployment.targetEnv = "container";
  };
}
```

2. nixops create -d containers containers.nix
3. nixops deploy -d containers

**Expected behavior**
NixOps containers should deploy as per 19.09

**Metadata**
Please run `nix run nixpkgs.nix-info -c nix-info -m` and paste the result.

 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.80, NixOS, 20.03.git.3600af2 (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.1`
 - channels(root): `"nixos-19.09.976.c75de8bc12c"`
 - nixpkgs: `/etc/nixos/nixpkgs`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by veprbl** [comment]
This probably belongs in https://github.com/NixOS/nixops/issues

**Comment by adrianparvino** [comment]
To anyone curious who goes through this: You'll need to revert 7cb100b6833.

**Comment by Ma27** [comment]
Fixed in ebb6e385eac53f74963c95ac25cc22b16d161177 / 7f1ba606ac632c50efd33c435aa59e69d4f58acc
